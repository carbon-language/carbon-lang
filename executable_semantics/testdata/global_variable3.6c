// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test type checking of global variable. Error expected.

var Int: flag = true;

fn main() -> Int {
  return 0;
}
