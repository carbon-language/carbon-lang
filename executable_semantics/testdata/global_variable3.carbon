// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

// Test type checking of global variable. Error expected.

var flag: Int = true;

fn main() -> Int {
  return 0;
}
