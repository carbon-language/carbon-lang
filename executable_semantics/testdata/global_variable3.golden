COMPILATION ERROR: executable_semantics/testdata/global_variable3.carbon:7: type error in initializer of variable
expected: i32
actual: Bool
EXIT CODE: 255
